spec/tapioca/cli/dsl_spec.rb

```diff
@@ -33,7 +33,7 @@ class DslSpec < CliSpec
           Loading DSL generator classes... Done
           Compiling DSL RBI files...
 
-          Wrote: #{outdir}/post.rbi
+                create  #{outdir}/post.rbi
```

Comment:
I assume we can't customize the "create" string 😕.

Reply:
Unfortunately, no. But in this case I think it's a bit better looking because it's at least consistent.